client/blocks/reader-join-conversation/style.scss

```diff
@@ -0,0 +1,69 @@
+.reader-join-conversation-dialog {
+	border-radius: 6px; /* stylelint-disable scales/radii */
+	width: 321px;
```

Comment:
Preferably, we should work with established values, ideally - with variables. That applies to the rest of the custom spacing, sizing and color styles in this PR.